src/app/no-overwritten-providers/no-overwritten-providers.component.spec.ts

```diff
@@ -1,10 +1,12 @@
-import { createHostComponentFactory } from '@netbasal/spectator';
+import { createHostComponentFactory, mockProvider } from '@netbasal/spectator';
 import { ComponentWithoutOverwrittenProvidersComponent } from './no-overwritten-providers.component';
+import { DummyService } from './dummy.service';
 
 describe('ComponentWithoutOverwrittenProvidersComponent', () => {
   describe('with options', () => {
-    let createHost = createHostComponentFactory({
-      component: ComponentWithoutOverwrittenProvidersComponent
+    const createHost = createHostComponentFactory({
```

Comment:
I'm not sure how this spec works before. There is a provider on the component's providers.